### PR TITLE
feat: add cookiebot

### PR DESCRIPTION
**Pull request by begonaalvarezd**
# Description of change

This PR aims to add CookieBot to the wiki, in order to automatically manage third party cookies.
Because we have to whitelist in cookiebot every single allowed deploy, I created a deploy that is already whitelisted https://iota-wiki-ruddy.vercel.app

Please note that Matomo has been updated to work with Cookiebot, so it would be great knowing if indeed the change works

## Links to any relevant issues

Closes https://github.com/boxfish-studio/iota-wiki/issues/4

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix
- Documentation Enhancement

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [ ] I have performed a self-review of my own changes
- [ ] I have made sure that added/changed links still work
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/common/pages/cookie-policy/index.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  useEffect(() => {
+
+    const appendCookiebotCdReport = new Promise((resolve, reject) => {
+      const script = document.createElement('script');
+      document.body.appendChild(script);
+      script.onload = resolve;
+      script.onerror = reject;
+      script.async = true;
+      script.src =
+        'https://consent.cookiebot.com/8f051d60-4ecb-41a0-abb9-4874fd999e4f/cdreport.js?referer=iota-wiki-fawn.vercel.app';
```
Comment (begonaalvarezd) [inline src/common/pages/cookie-policy/index.tsx:22]: before merging, this line has to be updated to the production URL 🙏🏼

**Review by begonaalvarezd — COMMENTED** [review]
Comment to myself 📔

File: static/js/matomo.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+var _paq = (window._paq = window._paq || []);
+/* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+_paq.push(['requireCookieConsent']);
+_paq.push(['trackPageView']);
+_paq.push(['enableLinkTracking']);
+(function () {
+  var u = 'https://matomo.iota-community.org/';
+  _paq.push(['setTrackerUrl', u + 'matomo.php']);
+  _paq.push(['setSiteId', '13']);
```
Comment (Dr-Electron) [inline static/js/matomo.js:9]: Can you change the id to 14 for testing?
```suggestion
  _paq.push(['setSiteId', '14']);
```

File: src/common/pages/cookie-policy/index.tsx (hunk 1 above)
Comment (Dr-Electron) [reply src/common/pages/cookie-policy/index.tsx:19]: We also need to change back the side id to `13` [here](https://github.com/iota-wiki/iota-wiki/blob/326a62a09026b5747c51a228dcc90d82f057095a/static/js/matomo.js#L9)

File: static/js/matomo-cookiebot.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      setTimeout(matomoWaitForTracker, 250);
+      waitForTrackerCount++;
+      return;
+    }
+  } else {
+    _paq.push(['requireConsent']);
+    window.addEventListener('CookiebotOnAccept', function (e) {
+      consentSet();
+    });
+    window.addEventListener('CookiebotOnDecline', function (e) {
```
Comment (Dr-Electron) [inline static/js/matomo-cookiebot.js:15]:
```suggestion
    window.addEventListener('CookiebotOnAccept', function () {
      consentSet();
    });
    window.addEventListener('CookiebotOnDecline', function () {
```

This should work too, right? And as `e` is not used anyway...
Not sure what's best practice in js

**Review by jlvandenhout — APPROVED** [review]
With the small changes Kevin proposed, LGTM.

**Pull request merged**